flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1447,6 +1460,35 @@ def test_resize_destroyed_volume(self):
         )
         self.assertEqual(exception.args, (volume.blockdevice_id,))
 
+    def test_foreign_cluster_volume(self):
```

Comment:
Discussed offline - should move into utility method and then add explicitly to Cinder and EBS test suites.